### PR TITLE
User profile branch

### PR DESCRIPTION
**Pull request by ogwuazors**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

### CODE REVIEW

File: config/env.js [hunk 1]
```diff
@@ -1,21 +1,20 @@
-'use strict';
```
Comment (jonkiky) [inline config/env.js:1]: I am not sure if changes on env.js is intense .

File: src/components/Auth/components/afterSignInComponent.js [hunk 2]
```diff
@@ -90,14 +142,21 @@ const styles = () => ({
   },
   paper: {
     background: '#465F96',
-    width: '120px',
+    // width: '120px',
```
Comment (jonkiky) [inline src/components/Auth/components/afterSignInComponent.js:145]: We can just delete this line.  No need to comment out.

**Review by jonkiky — CHANGES_REQUESTED** [review]

File: config/env.js (hunk 1 above)
Comment (ogwuazors) [reply config/env.js]: i think it was caused by eslint --fix

File: src/bento/siteWideConfig.js [hunk 3]
```diff
@@ -9,5 +9,5 @@ export default {
 
 export const loginRoute = '/login';
 export const requestAccessRoute = '/request';
-export const adminPortal = '/admin';
```
Comment (iksheth) [inline src/bento/siteWideConfig.js:12]: @ogwuazors Overwritten Item.

File: src/components/Auth/components/afterSignInComponent.js [hunk 4]
```diff
@@ -3,6 +3,9 @@ import { useSelector } from 'react-redux';
 import {
   Button, withStyles, Paper,
 } from '@material-ui/core';
+import IconButton from '@material-ui/core/IconButton';
```
Comment (iksheth) [inline src/components/Auth/components/afterSignInComponent.js:6]: @ogwuazors This was removed.

File: src/components/Auth/components/afterSignInComponent.js [hunk 5]
```diff
@@ -45,6 +48,15 @@ const AfterSignIn = ({
         {userName}
       </Button>
 
+      <IconButton
```
Comment (iksheth) [inline src/components/Auth/components/afterSignInComponent.js:51]: @ogwuazors This was removed.

File: src/components/Auth/loginComponent.js [hunk 6]
```diff
@@ -36,8 +36,6 @@ const IndexPage = ({ classes }) => {
     isSignedIn, email, firstName,
   } = useSelector((state) => state.login);
 
-  const userName = firstName || (email && email.substring(0, email.lastIndexOf('@')));
```
Comment (iksheth) [inline src/components/Auth/loginComponent.js:39]: @ogwuazors This was added in separate PR. It's overwritten section.

File: src/components/Auth/loginComponent.js [hunk 7]
```diff
@@ -46,7 +44,7 @@ const IndexPage = ({ classes }) => {
       <>
         { (isSignedIn) ? (
           <>
-            <AfterSignInComponent userName={userName} signoutLink={signOut} />
+            <AfterSignInComponent userName={firstName || email} signoutLink={signOut} />
```
Comment (iksheth) [inline src/components/Auth/loginComponent.js:47]: @ogwuazors This was added in separate PR. It's overwritten section.  It bringing up old changes.

File: src/components/Layout/LayoutView.js [hunk 8]
```diff
@@ -79,8 +81,6 @@ const Layout = ({ classes, isSidebarOpened }) => (
             <PrivateRoute path="/adminportal" access={['admin']} component={fakeAdminView} />
             <PrivateRoute path="/admin/edit/:id" access={['admin']} component={editUserController} />
             <PrivateRoute path="/admin/view/:id" access={['admin']} component={viewUserController} />
-            <PrivateRoute path="/admin" access={['admin']} component={adminController} />
```
Comment (iksheth) [inline src/components/Layout/LayoutView.js:82]: @ogwuazors This was added in separate PR. It's overwritten section.  It's removing new features.

File: src/pages/modelPage/modelPageView.js [hunk 9]
```diff
@@ -96,7 +96,7 @@ const ModelPage = ({ classes }) => {
             </div>
             <div className={classes.imgSection}>
               <TransformComponent>
-                <img src="https://cbiit.github.io/ctdc-model/model-desc/ctdc-model.svg" alt="ICDC schema" className={classes.img} />
+                <img src="//ctdc-model/model-desc/ctdc-model.svg" alt="ICDC schema" className={classes.img} />
```
Comment (iksheth) [inline src/pages/modelPage/modelPageView.js:99]: @ogwuazors is this URL correct?

**Review by iksheth — CHANGES_REQUESTED** [review]

**Comment by iksheth** [comment]
Closing This PR. Changes have been merged using another PR. #577

**Pull request closed without merging**